Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -122,13 +121,6 @@ verifyRC ::
   Flow DriverRCRes
 verifyRC isDashboard mbMerchant (personId, _, merchantOpCityId) req@DriverRCReq {..} = do
   person <- Person.findById personId >>= fromMaybeM (PersonNotFound personId.getId)
-  mbFleetOwnerId <- Redis.safeGet $ makeFleetOwnerKey vehicleRegistrationCertNumber
-  whenJust mbFleetOwnerId $ \fleetOwnerId -> do
-    isFleetDriver <- FDVAQ.findByDriverIdAndFleetOwnerId personId fleetOwnerId
-    case isFleetDriver of
-      Nothing -> throwError (InvalidRequest "Driver is not part of this fleet, add this driver to the fleet before adding a vehicle with them")
```

Comment:
These changes were only made because of the fleet adjustments. So, we only need to check them there, not here. The problem was that if the Redis key creation failed previously, the key would remain. This caused issues when drivers tried to add vehicles independently.